packages/terra-date-input/src/DateInput.jsx

```diff
@@ -827,7 +819,7 @@ class DateInput extends React.Component {
     const format = DateInputUtil.getDateFormat(this.props);
     const label = a11yLabel || this.props.intl.formatMessage({ id: 'Terra.date.input.labelDefault' });
     const dateFormatLabel = this.props.intl.formatMessage({ id: 'Terra.date.input.dateFormatLabel' });
-    const inputDate = moment(new Date(this.state.dateString)).format('dddd MMMM D YYYY');
+    const inputDate = moment(this.state.dateString).format('dddd MMMM D YYYY');
```

Comment:
what is the reason for removing the new operator here ?

Reply:
I have removed the 'new Date' instantiation because in PST timezone it is converting to one day less than the current date. So that it has been removed to fix this issue and now it is working fine.